evalml/tests/data_checks_tests/test_target_distribution_data_check.py

```diff
@@ -37,19 +37,17 @@ def test_target_distribution_data_check_no_y(X_y_regression):
 
 
 @pytest.mark.parametrize("target_type", ["boolean", "categorical", "integer", "double"])
-def test_target_distribution_data_check_unsupported_target_type(
-    target_type, X_y_regression
-):
-    X, y = X_y_regression
+def test_target_distribution_data_check_unsupported_target_type(target_type):
+    X = pd.DataFrame(range(5))
 
     if target_type == "boolean":
-        y = pd.Series([True, False] * 50)
+        y = pd.Series([True, False] * 5)
     elif target_type == "categorical":
-        y = pd.Series(["One", "Two", "Three", "Four"] * 25)
+        y = pd.Series(["One", "Two", "Three", "Four", "Five"] * 2)
     elif target_type == "integer":
-        y = pd.Series(np.random.randint(1, 25, 100))
+        y = [-1, -3, -5, 4, -2, 4, -4, 2, 1, 1]
     else:
-        y = np.random.normal(4, 1, 100)
+        y = [9.2, 7.66, 4.93, 3.29, 4.06, -1.28, 4.95, 6.77, 9.07, 7.67]
```

Comment:
Both these test blocks would previously throw errors 7-8 times out of a 100 runs. Even after changing the values to be more gaussian in distribution with larger sample sizes, they still failed 1-2 times out of the 100,000 simulations I ran. Randomness leads to a somewhat lognormal distribution every once in a while. I decided to replace these with constant smaller samples of a lognormal distribution so we don't have to worry about inconsistency again.

Reply:
Agreed!